data-processing/entities/definitions/commands/detect_definitions.py

```diff
@@ -377,6 +381,24 @@ class SymbolNickname(NamedTuple):
     nickname_indexes: List[int]
     nickname_ranges: List[CharacterRange]
 
+def get_filtered_symbol_nickname_pairs(
```

Comment:
Nit pick: maybe this could be more succinctly named the following?

```suggestion
def filter_symbol_nickname_pairs(
```